pipelines/models/lightgbm-iris/README.md

```diff
@@ -0,0 +1,23 @@
+# Iris model
+## Description
+This model classifies Iris flowers in order to distinguish between the three Irish species (Iris setosa, Iris virginica and Iris versicolor).
+
+## Dataset
+
+This model was trained using the well known [Iris](https://archive.ics.uci.edu/dataset/53/iris) dataset.
+
+## Test 
+### Run Seldon MLServer with IRIS model
+Execute the following command from the [pipelines](../../) folder:
+```
+docker run -d -u $(id -u) --rm -v ${PWD}/models:/opt/models:Z -p 8080:8080 -p 8081:8081 -p 8082:8082 -ti seldonio/mlserver:1.3.5-lightgbm mlserver start /opt/models/lightgbm-iris
```

Comment:
Ok I can do it, I need only to verify that all the provided options will still work for podman ;-)

Reply:
I can confirm that my verification was with `podman`

Reply:
@grdryn @LaVLaS I have to remove the `-u $(id -u)` because it is no more needed and it is causing the error `Error: OCI runtime error: crun: chown `/dev/pts/0`: Invalid argument`